bower.json

```diff
@@ -26,12 +25,19 @@
     "x-editable": "~1.5.1",
     "slickgrid": "~2.1.0",
     "reconnectingWebsocket": "#07669519f8",
+    "truncate": "https://github.com/pathable/truncate.git#5da408e2325dd965e0c94c354622ad54645824b9",
     "dropzone": "https://github.com/sloria/dropzone.git#accept-directory",
     "treebeard": "https://github.com/caneruguz/treebeard.git#4112b22be4333c9fce35249131632b6b2a37bb87",
     "jquery.cookie": "~1.4.1",
     "hgrid": "~0.2.10",
     "uri.js": "~1.14.1",
     "xhook": "~1.3.0",
+    "osf-panel": "https://github.com/caneruguz/osf-panel.git#develop",
+    "markdown-it": "~3.0.5",
+    "markdown-it-sanitizer": "~0.2.0",
+    "markdown-it-video": "https://github.com/brianjgeiger/markdown-it-video.git#b8b1cccb505a6773c30f8919a31b34092bac8e60",
+    "markdown-it-toc": "https://github.com/samchrisinger/markdown-it-toc.git#8c7fb8bfd25ba8b4fadd12aceb923f3066899743",
+    "jsdiff": "~1.2.2",
```

Comment:
Prefer npm distributions to bower components. There is one available for `jsdiff`. You should just have to move this to `package.json`.